packages/web3-shared/evm/hooks/useERC20TokenApproveCallback.ts

```diff
@@ -61,10 +61,10 @@ export function useERC20TokenApproveCallback(address?: string, amount?: string,
 
     const approveCallback = useCallback(
         async (useExact = false) => {
+            setTransactionState({
+                type: TransactionStateType.UNKNOWN,
+            })
```

Comment:
```suggestion
            setTransactionState({ type: TransactionStateType.UNKNOWN })
```

Reply:
Ahhh, that's endless "optimization", please comment by rules, if there is not rules, try to establishing rules. Otherwise, people don't know what to follow.